task/test-tests-scan

```diff
@@ -248,7 +249,21 @@ class TestTestsScan(unittest.TestCase):
         self.assertEqual(request["sha"], "abcdef")
         self.assertEqual(request["ref"], "abcdef")
         self.assertEqual(request["name"], "pull-1")
-        self.assertEqual(request["container"], "supertasks")
+        self.assertEqual(request["job"], {
+            "context": "fedora/nightly",
+            "repo": "project/repo",
+            "sha": "abcdef",
+            "slug": "pull-1-abcdef-20240102-030405-fedora-nightly",
+            "target": "stable-1.0",
```

Comment:
I think I might like to see a simulated case of a `main` run, ie:
 - `"target": None`
 - `"BASE_BRANCH"` envvar unset
 - `"report": {"title": f"Nightly tests failed on {image}", "labels": ["nightly"]}`

Reply:
I added a unit test for "direct SHA trigger" (aka nightly main run) to PR #5955. This spotted some nice bugs! Once that lands, I'll rebase. target and BASE_BRANCH behave as ordered, but there's no "report" here yet. We currently create that in tests-invoke, which is on the chopping block, so I take it we should set that directly in `tests-scan`. Also, let's drop the term "Nightly", we are allowed to trigger them while the sun is up, too :grin:

Reply:
Done.